src/test/java/de/tum/in/www1/artemis/util/GitUtilService.java

```diff
@@ -50,48 +51,41 @@
 
     private Git localGit;
 
-    public Repository initRepo() {
+    public void initRepo() {
         try {
             deleteRepos();
-            Git remoteGit = Git.init().setDirectory(new File(remoteRoot + "/" + remoteName)).call();
-            new File(remoteRoot + "/" + remoteName + "/" + FILES.FILE1).createNewFile();
-            new File(remoteRoot + "/" + remoteName + "/" + FILES.FILE2).createNewFile();
-            new File(remoteRoot + "/" + remoteName + "/" + FILES.FILE3).createNewFile();
+            Git remoteGit = Git.init().setDirectory(new File(remoteRepositoryRoot + "/" + remoteRepositoryName)).call();
+            new File(remoteRepositoryRoot + "/" + remoteRepositoryName + "/" + FILES.FILE1).createNewFile();
+            new File(remoteRepositoryRoot + "/" + remoteRepositoryName + "/" + FILES.FILE2).createNewFile();
+            new File(remoteRepositoryRoot + "/" + remoteRepositoryName + "/" + FILES.FILE3).createNewFile();
             remoteGit.add().addFilepattern(".").call();
             remoteGit.commit().setMessage("initial commit").call();
 
             // TODO: use a temp folder instead
-            localGit = Git.cloneRepository().setURI(System.getProperty("user.dir") + "/" + remoteRoot + "/" + remoteName + "/.git")
-                    .setDirectory(new File(repoRoot + "/" + repositoryName)).call();
-
-            FileRepositoryBuilder builder = new FileRepositoryBuilder();
-            builder.setGitDir(new java.io.File(repoRoot + "/" + repositoryName + "/.git")).readEnvironment() // scan environment GIT_* variables
-                    .findGitDir().setup();
-            localRepo = new Repository(builder);
-
-            builder = new FileRepositoryBuilder();
-            builder.setGitDir(new java.io.File(remoteRoot + "/" + remoteName + "/.git")).readEnvironment() // scan environment GIT_* variables
-                    .findGitDir().setup();
-            remoteRepo = new Repository(builder);
+            localGit = Git.cloneRepository().setURI(System.getProperty("user.dir") + "/" + remoteRepositoryRoot + "/" + remoteRepositoryName + "/.git")
+                    .setDirectory(new File(localRepositoryRoot + "/" + localRepositoryName)).call();
```

Comment:
Same here

Reply:
No, good catch. I think I just oberlooked it here. Will change it